src/WarcraftLegacies.Source/Quests/Zandalar/QuestGundrak.cs

```diff
@@ -1,39 +1,63 @@
-using MacroTools.FactionSystem;
+﻿using MacroTools.FactionSystem;
 using MacroTools.QuestSystem;
 using MacroTools.QuestSystem.UtilityStructs;
 using WarcraftLegacies.Source.Setup.Legends;
-using static War3Api.Common; 
+using static War3Api.Common;
 
 namespace WarcraftLegacies.Source.Quests.Zandalar
 {
+
+  /// <summary>
+  /// Capture <see cref="LegendNeutral.Gundrak"/> to unlock a new unit
+  /// </summary>
   public sealed class QuestGundrak : QuestData
   {
-    private static readonly int GundrakResearch = FourCC("R02Q");
-    private static readonly int WarlordId = FourCC("nftk");
-    private static readonly int TrollShrineId = FourCC("o04X");
-    
+    private static readonly int GundrakResearch = Constants.UPGRADE_R02Q_QUEST_COMPLETED_THE_DRAKKARI_FORTRESS_WARSONG;
```

Comment:
```suggestion
    private static readonly int GundrakResearch = Constants.UPGRADE_R02Q_QUEST_COMPLETED_THE_DRAKKARI_FORTRESS_WARSONG;
```
Might as well do this for other constants as well.

Reply:
I don't think my suggestion went through correctly here... I intended to suggest that you use `const` syntax.